packages/svelte/src/legacy/legacy-client.js

```diff
@@ -170,3 +171,72 @@ class Svelte4Component {
 export function run(fn) {
 	user_pre_effect(fn);
 }
+
+/**
+ * Function to mimic the multiple listeners available in svelte 4
+ * @param {EventListener[]} handlers
```

Comment:
Everything exported from a `/legacy` module should be deprecated

```suggestion
 * @deprecated
 * @param {EventListener[]} handlers
```